components/overflow-group/overflow-group-mixin.js

```diff
@@ -51,26 +45,27 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 				attribute: 'min-to-show',
 			},
 			/**
-			 * maximum amount of buttons to show
+			 * maximum amount of slotted items to show
 			 * @type {number}
 			 */
 			maxToShow: {
 				type: Number,
 				reflect: true,
 				attribute: 'max-to-show',
 			},
-			/**
-			 * @ignore
-			 */
-			mini: {
-				state: true
-			},
 			/**
 			 * @ignore
 			 */
 			openerType: {
 				type: String,
 				attribute: 'opener-type'
+			},
+			_chompIndex: {
```

Comment:
I put these back to how they were as I realized we can just send more useful parameters to `getOverflowContainer`. Can also put these back to being states though.